LeGO_LOAM: {
  ImageProjection: {
    mode : 0
  },
  LIDAR_sensor: {
    N_SCAN : 16,
    Horizon_SCAN : 1800,
    ang_res_x : 0.2,
    ang_res_y : 2.0,
    ang_res_bottom : 15.0,
    groundScanInd : 7
  }
}